src/utils/queryProcessing.tsx

```diff
@@ -0,0 +1,24 @@
+import { SerializedError } from '@reduxjs/toolkit';
+import { FetchBaseQueryError } from '@reduxjs/toolkit/query';
+
+class queryProcessing {
+  errorStatus(error: FetchBaseQueryError | SerializedError) {
+    if (
+      'data' in error &&
+      typeof error.data === 'object' &&
+      error.data !== null &&
+      'status' in error.data &&
+      typeof error.data.status === 'string'
+    ) {
+      localStorage.removeItem('sessionID');
```

Comment:
I'd name the LS key something like 'isAuthenticated'

Reply:
in the next PR LS key has already been renamed